doc/ccxt.pro.manual.rst

```diff
@@ -393,7 +393,7 @@ The CCXT Pro library currently supports the following 46 cryptocurrency exchange
      - coinex
      - `CoinEx <https://www.coinex.com/register?refer_code=yw5fz>`__
      - .. image:: https://img.shields.io/badge/1-lightgray
-          :target: https://github.com/coinexcom/coinex_exchange_api/wiki
+          :target: https://viabtc.github.io/coinex_api_en_doc
```

Comment:
we only need to update the .ts file, any other file is generated

Reply:
@carlosmiei the main link I wanted to update was in the README of the CCXT repo